standard/docs/en/schema/merging.md

```diff
@@ -1,30 +1,31 @@
-# Merging 
+# Merging
 
-In OCDS, merging involves combining individual [releases](../getting_started/releases_and_records.md) of data during a contracting process into a [record](../getting_started/releases_and_records.md) which provides an overview of the current state of that process. Including a versioned release as part of the merged record provides structured information on the history of the process: showing when updates were made, or values were changed. 
+An OCDS [record](../getting_started/releases_and_records.md) aggregates all the releases available for a contracting process at a given time, and optionally includes:
 
-<div class="example hint" markdown=1>
+* a compiled release, which expresses the current state of the contracting process, by showing only the most recent field values
+* a versioned release, which expresses all historical states of the contracting process, by showing all the field values over time
 
-<p class="first admonition-title">Worked Example</p>
+**Merging** is the process of combining individual releases into a compiled or versioned release, described in more detail below.
 
```

Comment:
As per my later comment about essential information, I suggest signposting the technical processes described later in detail.

The bullet points on lines 5 and 6 do a good job of explaining the differences between compiled and versioned releases. We should also indicate at a high level what the implementation of merging will look like.

I suggest something to the effect of "A compiled release is produced by taking the latest version of each field from the releases in a contracting process" and "A versioned release is produced by replacing a field's value with an array of versioned values, which should be in chronological order by `releaseDate`"

Reply:
Added bullet points in 50ee7f9